docs/quickstart.rst

```diff
@@ -172,6 +184,6 @@ resulting collections.
 
 .. code-block:: python
 
-    for collection in collection_search.collections():
-        print(collection.id)
+    for collection in collection_search.collections_as_dicts():
+        print(collection.get("id"))
```

Comment:
Why not use the pystac object?

Reply:
Alternately, I don't think we should have collection search in the quickstart until a public catalog supports it and works without a workaround like this.  I'd much rather have a good Filter example and an example of using an Extension.

Reply:
I used a CMR-STAC catalog instaed

Reply:
🫠 NASA gonna NASA